### DISCUSSION TITLE
Clue Solver

### DISCUSSION THREAD

**Discussion by SECI1** [whs/runekit#41] [General]
I got alt 1 to pop up at the top right bar, but when the pop-up of clue solver comes up, it keeps telling me that the capture settings need alteration. Yet, when I press the setting, I get a completely white screen for the clue solver application. Could you please help me out and let me know what I did wrong, or if you've faced the same problem, how did you fix it?

**Comment by bjuno** [comment]
Hey there, any update on this one? I am getting the same issue.

**Reply by SECI1** [reply]
Hey, I got it working. The problem was the size of the screen, so when I connected it to my monitor it started solving my hard clues. Also, when the puzzle clue comes up, there is no red layover to helps you know where to click. But there is a voice-over option which works well'ish but its faster then manually solving it just takes some time. done a 100 hard clues with this and its working okay.

**Reply by bjuno** [reply]
So you needed to change the RS window or monitor resolution?
